docs/site/Decorators.md

```diff
@@ -132,56 +132,181 @@ accordingly or do a composition of them:
   For more usage, refer to [Routing to Controllers](controllers.htm#routing-to-controllers)
 
 ### Parameter Decorator
+ 
+  Syntax: see [API document](https://github.com/strongloop/loopback-next/blob/master/packages/openapi-v3/src/parameter-decorator.ts#L18-L29)
```

Comment:
Did you perhaps mean `API documentation`?

Also, could you please point to http://apidocs.loopback.io instead of TypeScript source?

If we decide to keep the link to a ts file, then please make sure the link points to a specific revision, not the `master` branch. Otherwise the link will point to wrong line numbers soon, as the code will evolve.

Reply:
@bajtos oops a typo, there is a bug of `ts` doc parsing function:

Given a function and namespace with the same name, it only parses the namespace, but could not detect the function. That's why I point it to the code as a workaround.

Reply:
Interesting...seems people have different opinions on the link's branch. I use "a specific version" before but Kevin corrected me to use master....personally I agree with "a specific version".

Appreciate more opinion from team @strongloop/lb-next-dev , for the code links in documentation, would you point them to `master` branch or a specific version?

Reply:
It is often that the line numbers change over time, so I'd avoid pointing to the source code, no matter which branch it is pointing to.    My question to you, @jannyHou, would be.. what do we want to show to the users that we cannot show in a more static document, like API docs?

Reply:
@dhmlau My link points to the tsdoc of source code :( I agree that this is the last choice...while could not think of a better idea as workaround